### DISCUSSION TITLE
Is a multi-select variable possible?

### DISCUSSION THREAD

**Discussion by leethobbit** [chhoumann/quickadd#669] [Q&A]
Hello, I've used QuickAdd for years now and love the utility of it.  However, one use case I've never been able to figure out is if it can handle some sort of multi-select.

For example, during Template note generation, I'd love to be able to present a bunch of possible tags - you could then select any number (or none) of them.  Right now, the way I work around this is to list the options in the name of the variable like so:
`{{VALUE:Tags? #writing #work #personal #travel #okr}}`

You then have to manually type out the tags you want. It works OK for this example - there are only 5 tags.  However, I have been rebuilding my templates and would really like to present something much larger - say, 15 different options and any number can be chosen.

Is there a better way to accomplish this? If not, is this something that could be implemented?

Thank you!

**Comment by chhoumann** [comment]
Coming back to this one very late - sorry about that.

Multi-select shipped in QuickAdd 2.14.0:

```
tags: {{VALUE:writing,work,personal,travel,okr|multi}}
```

opens a multi-select picker where you can pick any number of the options. Inside front matter it writes a real YAML List; in the note body it renders comma-separated. Variants:

- `|multi|custom` adds a text box so you can also enter values not in the list
- `|multi:linklist` wraps each pick as a wikilink
- it combines with `|optional`, `|label:`, and `|name:` (reuse the same selection elsewhere in the template)

Your 15-option tag picker is exactly the case this was built for - no more typing tags out by hand.

Docs: https://quickadd.obsidian.guide/docs/FormatSyntax#value-multi

Closing as resolved.
